File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        int arr1n = 0;
        int arr2n = 0;
        if(arr1.length != arr2.length){
            if(arr1.length > arr2.length){
                return 1;
            }
            else 
                return -1;
        }
        else if(arr1.length == arr2.length){
            for(int i = 0; i < arr1.length; i++){
                arr1n += arr1[i];
            }
            for(int i = 0; i < arr2.length; i++){
                arr2n += arr2[i];
            }
            if(arr1n > arr2n){
                return 1;
            }
            if(arr1n < arr2n){
                return -1;
            }
            else return 0;
        }
        return answer;
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    some1 = 0
    some2 = 0
    if len(arr1) > len(arr2):
        return 1
    elif len(arr2) > len(arr1):
        return -1
    else:
        for i in range(len(arr1)):
            some1 += arr1[i]
            some2 += arr2[i]
        if some1 > some2:
            return 1
        elif some2 > some1:
            return -1
    return 0